sklearn/ensemble/_forest.py

```diff
@@ -415,8 +415,11 @@ def _validate_X_predict(self, X):
     @property
     def feature_importances_(self):
         """
-        Return the feature importances (the higher, the more important the
-           feature).
+        The impurity-based feature importances.
```

Comment:
Please add a blank line after the first sentence (only for the functions, no need to do that for the attribute descriptions)